capstone-frontend/src/components/SearchBookModal.jsx

```diff
@@ -179,7 +173,11 @@ export class SearchBookModal extends Component {
                         {this.state.addedBooksIDs.includes(book.id) ?
                           <Button className="my-5" variant="danger" onClick={() => this.removeBookFromList(book)}>Remove Book</Button>
                           :
-                          <Button className="my-5" onClick={() => this.addBookToList(book)}>Add to Booklist</Button>}
+                          this.props.type === "club" ?
+                            <Button className="my-5" onClick={async () => this.clubSubmit(book.id)}>Add</Button>
```

Comment:
nit: Don't think this needs to be `async`

Reply:
I'll fix this.